src/client/redux/shopping/operation.js

```diff
@@ -0,0 +1,40 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const fetchShoppingList = createAsyncThunk(
+  'shoppingList/fetchAll',
+  async (_, thunkAPI) => {
+    try {
+      const response = await axios.get('/shopping-list');
+      return response.data;
```

Comment:
same here. but if you want to handle response in data field implement that logic on backend

Reply:
So should ale responses be done as that?